python/mxnet/log.py

```diff
@@ -61,14 +61,40 @@ def format(self, record):
         return super(_Formatter, self).format(record)
 
 def getLogger(name=None, filename=None, filemode=None, level=WARNING):
-    """Get customized logger.
+    """Gets a customized logger.
 
-    Args:
-        name: Name of the logger.
-        level: Level to log.
+    Parameters
+    ----------
+    name: str, optional
+        Name of the logger.
+    filename: str, optional
+        The filename to which the logger's output will be sent.
+    filemode: str, optional
+        The file mode to open the file (corresponding to `filename`),
+        default is 'a' if `filename` is not ``None``.
+    level: int, optional
+        The `logging` level for the logger.
+        See: https://docs.python.org/2/library/logging.html#logging-levels
 
-    Returns:
-        A logger.
+    Returns
+    -------
+    Logger
+        A customized `Logger` object.
+
+    Example
+    -------
+    >>> from mxnet.log import getLogger
+    >>> logger = getLogger("Test")
+    >>> logger.warn("Hello World")
+    W0505 00:29:47 3525 <stdin>:<module>:1] Hello World
+    >>> import logging
+    >>> logger = getLogger("Test2", level=logging.WARNING)
+    >>> logger.warn("Hello World")
+    W0505 00:30:50 3525 <stdin>:<module>:1] Hello World
+    >>> logger.debug("Hello World")
```

Comment:
These are 3 different examples.
separate them with comments explaining what each example is trying to show